packages/grafana-ui/src/components/Table/TableNG/Cells/SparklineCell.tsx

```diff
@@ -112,7 +114,9 @@ export const SparklineCell = (props: SparklineCellProps) => {
   return (
     <div className={styles.cellContainer}>
       {valueElement}
-      <Sparkline width={width - valueWidth} height={25} sparkline={sparkline} config={config} theme={theme} />
+      <TableCellLinkWraper field={field} rowIdx={rowIdx}>
+        <Sparkline width={width - valueWidth} height={25} sparkline={sparkline} config={config} theme={theme} />
+      </TableCellLinkWraper>
```

Comment:
I like how simple it is to decide "this is the part of this cell I want to wrap for linking and datalink/action tooltip. this is really easy to use and plugs nicely with JSX imo.